sample/EhealthAfrica/event

```diff
@@ -1,60 +1,59 @@
-    "background_image": "/images/background.jpg", 
-    "logo": "/images/logo.jpg", 
-    "organizer_name": "eHealth Africa", 
-    "organizer_description": "eHealth Africa is one of the leading providers of digital health information systems in West Africa. The software we built ranges from mobile applications to help contain the Ebola outbreak, dashboards for tracking of vaccinations and new Polio cases in order to make Africa a Polio-free continent, emergency call center software to comprehensive health and disease information systems for whole countries.", 
-    "social_links": [
-        {
-            "id": 360, 
-            "link": "https://www.facebook.com/EHealthAfrica", 
-            "name": "Facebook"
+{   
+    "social-links": [{
+                    "id": 360, 
+                    "link": "https://www.facebook.com/EHealthAfrica", 
+                    "name": "Facebook"
         }, 
         {
-            "id": 361, 
-            "link": "https://twitter.com/eHealth_africa", 
-            "name": "Twitter"
+                    "id": 361, 
+                    "link": "https://twitter.com/eHealth_africa", 
+                    "name": "Twitter"
```

Comment:
@ParthS007 this indentation is incorrect.

Reply:
Thanks @agbilotia1998 , will do the changes